packages/modules/atlas-core/scripts/build.ts

```diff
@@ -40,6 +43,43 @@ async function main(): Promise<void> {
 
         await copyStylesAndAssets(mode === "start", outputDir);
     }
+
+    await buildAndCopyWidgets(mode, ["feedback-native"], outputDir);
```

Comment:
Why do we suddenly have to include `feedback-native`?

Reply:
Because it was forgotten before 😬